Dockerfile.dapper

```diff
@@ -17,7 +17,7 @@ ARG DAPPER_HOST_ARCH
 ENV ARCH $DAPPER_HOST_ARCH
 
 RUN zypper -n rm container-suseconnect && \
-    zypper -n install go1.16 git curl docker gzip tar wget zstd squashfs xorriso awk
+    zypper -n install go1.16 git curl docker gzip tar wget zstd squashfs xorriso awk mtools dosfstools
```

Comment:
Can you explain why these two tools are required?
I feel like we need to record the reason when adding tools otherwise it will become hard to remove afterwards

Reply:
Sure! Rebased and add the reason in the comment.